include/dicom-checksum.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include <core.h>
+
+class DicomChecksum {
+public:
+    static std::unordered_map<std::string, std::tuple<std::string, int>> checksum_map;
```

Comment:
Does this need to be public? I haven't inspected other source files but the methods of the class access it, so I'd presume nothing else needs to modify its state.